lib/static/components/details.jsx

```diff
@@ -10,7 +10,8 @@ export default class Details extends Component {
         title: PropTypes.oneOfType([PropTypes.element, PropTypes.string]).isRequired,
         content: PropTypes.oneOfType([PropTypes.func, PropTypes.string, PropTypes.element, PropTypes.array]).isRequired,
         extendClassNames: PropTypes.oneOfType([PropTypes.array, PropTypes.string]),
-        onClick: PropTypes.func
+        onClick: PropTypes.func,
+        asHtml: PropTypes.bool,
```

Comment:
Ability to pass raw html is needed in order to display our html, created from ansi